letsencrypt/auth_handler.py

```diff
@@ -113,6 +113,18 @@ def _solve_challenges(self):
                     cont_resp = self.cont_auth.perform(self.cont_c)
                 if self.dv_c:
                     dv_resp = self.dv_auth.perform(self.dv_c)
+                    for achall, response in zip(self.dv_c, dv_resp):
+                        if isinstance(response, challenges.HTTP01Response):
+                            if not response.simple_verify(
+                                achall.chall, achall.domain,
```

Comment:
@pde, I feel like a robot here - would you mind turning back on relevant settings in the pylint configuration?